### DISCUSSION TITLE
关于国际化

### DISCUSSION THREAD

**Discussion by luchong0813** [HandyOrg/HandyControl#920] [Q&A] [answered]
编译时会在目录下生成国际化多语言的包，可能时默认开启了国际化，但我并不需要，请问如何关闭，以达到不让他在编译时自动生成这些包
![image](https://user-images.githubusercontent.com/46243214/126732446-701b2198-7d29-41dd-8e0e-663c281b9e0f.png)

**Comment by ghost1372** [comment]
Copy this code to csproj file

```<SatelliteResourceLanguages>en;fa</SatelliteResourceLanguages>```

**Reply by luchong0813** [reply]
不行，还是会生成这些语言文件夹
![image](https://user-images.githubusercontent.com/46243214/126743991-f49e2839-c28f-4271-be15-b38faf348481.png)

**Reply by ghost1372** [reply]
you have multiple platform and you wrote ```<SatelliteResourceLanguages>en;fa</SatelliteResourceLanguages>```

in Debug configuration so make sure you are building debug configuration
i always use this code and worked fine
https://github.com/HandyOrg/HandySub/blob/2535bd767a733c9165d5df5850638b64b3f01a7c/src/HandySub/HandySub.csproj#L15

**Reply by luchong0813** [reply]
你这个使用的是.net 5.0  而我使用的是.net framework4.5.2

**Reply by ghost1372** [reply]
if this way not worked, you can use visual studio PostBuild and after building project vs automatically delete your folders
`RD /S /Q "Full Path of Folder"`

image is just a demo not related to answer
![image](https://user-images.githubusercontent.com/9213496/126748249-4fbc6762-dd27-47f3-8d1d-58d9fc77ebc5.png)

**Reply by luchong0813** [reply]
通过编译HandyControl源代码 ，将Lang里面的其他资源文件删掉 然后编译 就可以了

**Reply by ghost1372** [reply]
there is an issue about this
https://github.com/HandyOrg/HandyControl/issues/626
but there is no update yet

**Reply by luchong0813** [reply]
这个解决办法应该只适用于.NET Core/5.0+

**Comment by NaBian** [comment, marked as answer]
复制一份 sln 文件，打开项目，删除语言包。
